tsconfig.build.json

```diff
@@ -11,6 +11,9 @@
 		},
 		"types": ["node"],
 		"declarationMap": false,
-		"sourceMap": false
+		"moduleResolution": "Node16",
```

Comment:
Minor note: should this be `node16`?

https://www.typescriptlang.org/tsconfig/#moduleResolution

Reply:
yes. it means it is generating commonjs that is friendly with the esm implementations in node 16 and later

perhaps it would be better to generate esm code here, but this is working so I don't see a reason to change it